v1/README_Cloud_BigQuery_to_GCS_TensorFlow_Records.md

```diff
@@ -22,17 +22,17 @@ on [Metadata Annotations](https://github.com/GoogleCloudPlatform/DataflowTemplat
 
 ### Required parameters
 
-* **readQuery** : A BigQuery SQL query that extracts data from the source. For example, select * from dataset1.sample_table.
-* **outputDirectory** : The top-level Cloud Storage path prefix to use when writing the training, testing, and validation TFRecord files. Subdirectories for resulting training, testing, and validation TFRecord files are automatically generated from `outputDirectory`. For example, `gs://mybucket/output/train` (Example: gs://mybucket/output).
+* **readQuery** : A BigQuery SQL query that extracts data from the source. For example, `select * from dataset1.sample_table`.
+* **outputDirectory** : The top-level Cloud Storage path prefix to use when writing the training, testing, and validation TFRecord files. Subdirectories for resulting training, testing, and validation TFRecord files are automatically generated from `outputDirectory`. (Example: gs://mybucket/output).
```

Comment:
@liferoad We have "example" fields defined for some of the parameters and it is generated automatically as "(Example: <>)" based on : https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/plugins/core-plugin/src/main/java/com/google/cloud/teleport/plugin/model/ImageSpecParameter.java#L654

I have tested by changing it to `this.setHelpText(this.getHelpText() + " For example, `" + example + "`");` so that the example gets generated in the readme as per our expectation like above `For example, `gs://mybucket/output/train`.`

This change will fix the example part in many places.